TFUtil.py

```diff
@@ -72,6 +72,7 @@ def __init__(self, name,
                available_for_inference=True,
                auto_create_placeholders=False,
                vocab=None,
+               unknown_label = "UNK",
```

Comment:
Please remove this. This is redundant. You can already pass it via the `vocab` argument. There is no need for an additional parameter.

Reply:
(Btw, also this code formatting is not PEP conform. There should not be whitespaces around the `=`. Also it's inconsistent to all the other code.)

Reply:
This is not redundant.  Otherwise I can't specify the vocabulary as file directly in the config file, I would need to load it somewhere else. I need it to be loaded by this class, and for this I need the unknown_label which may be different from the default one.

Reply:
Sure you can. Why do you think you can not?

Just like `... vocab={'vocab_file': ..., 'unknown_label': ...) ...`. Just see the code, and the documentation for the argument `vocab`. It can also be a `dict`.

Reply:
Sorry, I am not so good in python. Yes, it works this way, but the specification in the config file is already pretty complicated. Ideally, it should be as simple as possible, just name-value pairs without nesting. But I will revert this anyway, it is not too late, we haven't decided to freeze the config file conventions.

Reply:
I prefer it to be explicit and cleanly structured. The `Data` class should not know anything about the unknown label. This is specific to `Vocabulary`. Otherwise, imagine that other things get added to `Vocabulary`. It would be a nightmare to then add all further options also to lots of other unrelated classes.